src/RenderWebGL.js

```diff
@@ -1316,21 +1316,26 @@ class RenderWebGL extends EventEmitter {
 
         const dx = x - drawable._position[0];
         const dy = y - drawable._position[1];
+        let aabb;
 
-        const aabb = drawable.getFastBounds();
-        const inset = Math.floor(Math.min(aabb.width, aabb.height) / 2);
+        if (drawable._skin.typeOfSkin === 'BITMAP') {
+            aabb = drawable.getAABB();
+        } else {
+            aabb = drawable.getFastBounds();
+        }
```

Comment:
I'd prefer to avoid making decisions externally based on types. How would you feel about delegating this to a method on the skin? For example, `getFenceBounds` could call (or even _be_) `getAABB` for bitmaps and `getFastBounds` elsewhere.

Reply:
@cwillisf that makes sense to me! I'll make that change now